From: Simon McVittie <smcv@debian.org>
Date: Mon, 6 Sep 2021 11:11:32 +0100
Subject: t/python-prefix.sh: Respect PYTHON environment variable

Otherwise, this test will fail on operating systems that have python3
but not python (implicitly Python 2) in the PATH.

The test initialization in t/ax/*.sh sets PYTHON whenever we have
required=python, so we can assume it is set, and do not need to fall
back to calling python if it is unset.

Bug-Debian: https://bugs.debian.org/992416
Signed-off-by: Simon McVittie <smcv@debian.org>
---
 t/python-prefix.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/python-prefix.sh b/t/python-prefix.sh
index 3179d42..5c70d96 100755
--- a/t/python-prefix.sh
+++ b/t/python-prefix.sh
@@ -45,7 +45,7 @@ echo-python-exec-prefix:
 	@echo $(PYTHON_EXEC_PREFIX)
 END
 
-py_version=$(python -c 'import sys; print("%u.%u" % sys.version_info[:2])')
+py_version=$($PYTHON -c 'import sys; print("%u.%u" % sys.version_info[:2])')
 py_inst_site=inst/lib/python$py_version/site-packages
 py_instexec_site=instexec/lib/python$py_version/site-packages
 
